test/test_videoapi.py

```diff
@@ -76,8 +76,14 @@ def test_frame_reading(self, test_video):
 
                 # compare the frames and ptss
                 for i in range(len(vr_frames)):
-                    assert av_pts[i] == vr_pts[i]
-                    torch.test.assert_equal(av_frames[i], vr_frames[i])
```

Comment:
This was added in the wrong section so reverted the commit. My comment regarding why we cannot make the tests strict from read_from_memory still holds